docs/docker/upgrading/index.md

```diff
@@ -2,8 +2,8 @@
 
 The standard Pi-hole customization abilities apply to this docker, but with docker twists such as using docker volume mounts to map host stored file configurations over the container defaults. However, mounting these configuration files as read-only should be avoided. Volumes are also important to persist the configuration in case you have removed the Pi-hole container which is a typical docker upgrade pattern.
 
-!!! tip "Always Read The Release Notes!"
-    Ensure you read the release notes for both the Docker Release and the main Pi-hole component releases. This will help you avoid common problems due to known issues with upgrading or newly required arguments or variables
+!!! warning "Always Read The Release Notes!"
+    Ensure you read the release notes for both the Docker Release and the main Pi-hole component releases. This will help you avoid common problems due to known issues with upgrading or newly required arguments or variables. The Release notes can be found in the Respective Repository for [Core](https://github.com/pi-hole/pi-hole/releases), [FTL](https://github.com/pi-hole/FTL/releases), [Web](https://github.com/pi-hole/web/releases) and [Docker](https://github.com/pi-hole/docker-pi-hole/releases).
```

Comment:
```suggestion
    Ensure you read the release notes for both the Docker release and the main Pi-hole component releases. This will help you avoid common problems due to known issues with upgrading or newly required arguments or variables. The release notes can be found in the respective repository for [Core](https://github.com/pi-hole/pi-hole/releases), [FTL](https://github.com/pi-hole/FTL/releases), [Web](https://github.com/pi-hole/web/releases) and [Docker](https://github.com/pi-hole/docker-pi-hole/releases).
```